src/jobs/NotificationTrigger/pushNotificationSender.js

```diff
@@ -18,6 +17,7 @@ const sendPushNotification = async (notification) => {
 }
 
 const makeRequest = async (data = {}) => {
+	data.direct_book_ok = true;
```

Comment:
It is a good idea, but seems it requires some dependency in the app.
https://firebase.google.com/docs/cloud-messaging/android/receive#receive_fcm_messages_in_direct_boot_mode
@ShivaAryal